src/cmd/graphql.go

```diff
@@ -169,13 +172,6 @@ func init() {
 	graphqlCmd.Flags().StringArrayP("field", "f", nil, "Add a variable in `key=value` format")
 }
 
-func handleErr(msg string, err error) {
```

Comment:
We can add `cobra.CheckErr()` once here

Reply:
@davidbloss restored that function and removed all mentions of `errors` package. Now using `fmt.Errorf` everywhere to make errors, which is the idiomatic way.